### ISSUE TITLE
A11y_ML .NET Model Builder_ Data_ Screen Reader: Narrator/NVDA are not reading the label "Data Source Type"

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1118]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #Benchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Version 6.0

### Screen Readers:
Narrator; NVDA

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Text Classification" scenario and select the scenario
6. Tab till "Next Step" button and press enter
7.verify whether Screen readers are reading the label "Data Source Type" or not

### Actual Result:
Narrator/NVDA are not reading the label "Data Source Type". When focus comes to "File" radio button screen readers are reading as "File .csv .tsv .txt radio button selected"

### Expected Result:
When focus comes to first radio button screen readers should first read the label of the radio buttons group and then the radio button name. Screen readers should read as "Data Source Type   File .csv .tsv .txt radio button selected"       

### User Impact:
Screen Reader users will not get to know what are these radio buttons about if screen reader doesn't read the label

### Attachment for Reference
[1118_ML .NET Model Builder_ Narrator is not reading the radio buttons label.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607020/1118_ML.NET.Model.Builder_.Narrator.is.not.reading.the.radio.buttons.label.pptx)

[1118_ML .NET Model Builder_ NVDA is not reading the radio buttons label.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5607096/1118_ML.NET.Model.Builder_.NVDA.is.not.reading.the.radio.buttons.label.pptx)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the given build and the issue is fixed. Hence closing the bug
Please find the below screen shots for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (52)](https://user-images.githubusercontent.com/51435580/108031118-46aa5b00-7056-11eb-9d8e-ae344a0af132.png)
![Screenshot (51)](https://user-images.githubusercontent.com/51435580/108031111-44e09780-7056-11eb-8d08-9aaa7f1baf46.png)

**Comment by poojapatil36** [comment]
Github tag: #Closed;
